app/Http/Controllers/Entity/RelationController.php

```diff
@@ -204,7 +204,13 @@ public function edit(Entity $entity, Relation $relation)
     public function update(StoreRelation $request, Entity $entity, Relation $relation)
     {
         $this->authorize('update', $entity->child);
-        $data = $request->only(['target_id', 'attitude', 'relation', 'colour', 'is_star', 'two_way', 'visibility_id']);
+        $data = $request->only(['target_id', 'attitude', 'relation', 'colour', 'is_star', 'two_way', 'visibility_id', 'unmirror']);
```

Comment:
Seems cleaner to skip adding `unmirror`, and simply test `if ($request->has('unmirror')`, and skip the `unset()`